.storybook/helpers/files.data.ts

```diff
@@ -62,3 +62,14 @@ export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
     .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
     .join('');
 }
+
+export function getIconTreeNodeMarkup(files: File[] = filesRoot) {
+  return files
+    .map(
+      file =>
+        `<clr-tree-node><cds-icon [attr.shape]="'${file.files ? 'folder' : 'file'}'"></cds-icon>${file.name}${
+          file.files ? getIconTreeNodeMarkup(file.files) : ''
+        }</clr-tree-node>`
+    )
```

Comment:
```suggestion
    .map(
      file => `
        <clr-tree-node>
          <cds-icon [attr.shape]="'${file.files ? 'folder' : 'file'}'"></cds-icon>
          ${file.name}
          ${file.files ? getIconTreeNodeMarkup(file.files) : ''}
        </clr-tree-node>
      `
    )
```

This is more readable.

Reply:
This might be a moot issue after my other comment. https://github.com/vmware-clarity/ng-clarity/pull/1283/files#r1508134168

Reply:
Formatted the above to make it more readable as suggested.